### PR TITLE
Fix export of calculated WP to PN (fix #10876 )

### LINKED ISSUE #10876: Export of calculated WP to PN not working
@murggel 
I tried to test PR #10849 as follows:
- Took a traditional cache
- Added a reference point with calculated coordinates as follows:
N51 AB.139 E007 CD.529
A=t
B=f
C=u
D=r
f=3
r=4
t=5
u=2

This results in N51 53.139 E007 24.529

- If I backup the waypoints to the PN it adds only:
```
{c:geo-start}
@Markierung 1 (W)
{c:geo-end}
```

It seems not working even not if I ommit some values for variable to have an incomplete formula.

Tested with (to be release) 2021.06.06 as well as current NB.

### PR DESCRIPTION
**Pull request by murggel**
## Description

export coordinates to PN, if formula has not PLAIN-format.
fix regular expression to fit to user note with newline and ' as end of formula

## Related issues

fix #10876

### CODE REVIEW

File: main/src/cgeo/geocaching/calculator/FormulaParser.java [hunk 1]
```diff
@@ -199,7 +199,7 @@ public final ResultWrapper parse(@NonNull final String text, @NonNull final Geop
      */
     private static final class MinDecFormulaParser extends AbstractLatLonFormulaParser {
         //                                                        (    2    )                                      (      3      )
-        private static final String STRING_MINDEC = "\\s*(" + COORD_FORMULA_PATTERN_STRING + ")[°\\s]+(" + COORD_FORMULA_PATTERN_STRING + "\\." + COORD_FORMULA_PATTERN_STRING + ")['′\\s]?";
+        private static final String STRING_MINDEC = "\\s*(" + COORD_FORMULA_PATTERN_STRING + ")[°\\s]+(" + COORD_FORMULA_PATTERN_STRING + "\\." + COORD_FORMULA_PATTERN_STRING + ")['′]?[\\s]?";
```
Comment (murggel) [inline main/src/cgeo/geocaching/calculator/FormulaParser.java:202]: Should the last part of the regExpr be `[\\s]*` instead of `[\\s]? `?

Comment (bekuno) [reply main/src/cgeo/geocaching/calculator/FormulaParser.java:202]: What do you want to cover?

Square brackets meen caracter classes. And question mark an optional sign. 
See eg. https://www.hosteurope.de/blog/regex-grundlagen-arbeiten-mit-regulaeren-ausdruecken/.

Comment (murggel) [reply main/src/cgeo/geocaching/calculator/FormulaParser.java:202]: I think there is no need to restrict the last whitespace to be only zero or one.

**Comment by murggel** [comment]
Ready for review

**Comment by Lineflyer** [comment]
Can someone review and merge here?

Afterwards I could prepare the planned bugfix release.

**Comment by eddiemuc** [comment]
Looks good, thanks @murggel for the contribution!

**Pull request merged**